### ISSUE TITLE
Allow skipping lines matching regex/prefix for the `max-len` checker

### ISSUE THREAD

**Issue by pmalek** [godoc-lint/godoc-lint#80]
### Problem statement

Using `godoclint` for files that require annotations (e.g. Kubernetes CRD source files) yields the following errors:

```
api/konnect/v1alpha2/konnect_extension_types.go:128:1: godoc line is too long (205 > 127) (godoclint)
// KonnectExtensionClientAuth contains the configuration for the client authentication for the DataPlane.
^
```

This is triggered on the following code:

```
// KonnectExtensionClientAuth contains the configuration for the client authentication for the DataPlane.
// At the moment authentication is only supported through client certificate, but it might be extended in the future,
// with e.g., token-based authentication.
//
// +kubebuilder:validation:XValidation:rule="self.certificateSecret.provisioning == 'Manual' ? has(self.certificateSecret.secretRef) : true",message="secretRef must be set when provisioning is set to Manual."
// +kubebuilder:validation:XValidation:rule="self.certificateSecret.provisioning == 'Automatic' ? !has(self.certificateSecret.secretRef) : true",message="secretRef must not be set when provisioning is set to Automatic."
type KonnectExtensionClientAuth struct {
```

Since the contents of the offending lines is not printed it cannot be exclude with `golangci-lint` rules or with `godoclint` configuration.

### Proposed solution

Add a regex (array?) config field which will exclude lines from causing errors if they match said regex(es).

**Comment by babakks** [comment]
Thanks for the feedback, @pmalek! 🙏

I agree that `max-len` needs some configuration to skip specific lines. I'll probably add a configuration options to accept an array of regexps. Please note that for the changes to be available they'll also need to be applied in Golangci-lint, for which I'll open a PR.
